datastore-v1-proto-client/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -29,5 +29,55 @@
     "allPublicMethods":true,
     "allDeclaredConstructors" : true,
     "allPublicConstructors" : true
-  }
+  },
+  [
```

Comment:
Ah looks like we have an extra `[ ]` that's causing the graalvm job failure.

Reply:
😆 good catch, thanks!